tests/dbErrorHandlers.test.js

```diff
@@ -0,0 +1,49 @@
+const dbErrorHandlers = require('../src/dbErrorHandlers.js')
+const { messages: errMessages, names: errNames, codes: errCodes } = require('../src/constants/errors.json')
+const { buildNextSpyTest, buildBoomSpyTest } = require('./helpers/index.js')
+
+const tape = require('tape')
+
+tape('dbErrorHandlers.custom with non custom error', t => {
+  buildNextSpyTest(t, dbErrorHandlers.custom, 'test error message', 'not a custom error')
```

Comment:
Okay,

I think:
```js
callErrorMiddleware = errorMiddleware => (error) => {
  const resSpy = 
  const nextSpy = 
  errorMiddleware(error, undefined, resSpy, nextSpy)
}
```

Then in your tests
```js
test('testing something', t => {
  const callMiddleware = callErrorMiddleware(middlewareToTest)

  const errorToTest = some error
  const { resSpy, nextSpy } = callMiddleware(errorToTest)

  // now, in the test function, we can make assertions in plain English
})
```

Okay that is very rough, but the idea is that you do not really need to look in the helpers to understand what assertions are being made. The helper should have names that makes it obvious what they are doing, and the assertions want to live in the test.

If you want to make helpers that make assertions, the most you want to be doing is having functions that are called in tests like
```js
assertSomethingRepetive(t, actual, expected, msg)
```